packages/bpk-component-slider/src/BpkSlider.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @skyscanner/rules/forbid-component-props */
```

Comment:
This is needed here as a way to set styles on the external component

Reply:
Here we can add an exclusion to our linter so its centralised in one place, see here for examples: https://github.com/Skyscanner/backpack/blob/main/.eslintrc#L139